transport/node.go

```diff
@@ -0,0 +1,46 @@
+package transport
+
+import (
+	"go.uber.org/atomic"
+	"log"
+	"scylla-go-driver/frame"
+)
+
+// Dummy items used because they are still not incorporated to main.
+
+type ConnPool interface{}
+
+func InitNodeConnPool(addr string, cfg ConnConfig) *ConnPool {
+	return (*ConnPool)(nil)
+}
+
+type nodeStatus = atomic.Bool
+
+const (
+	statusDown = false
+	statusUP   = true
+)
+
+type Node struct {
+	addr       string
+	datacenter string
+	rack       string
+	tokens     []byte // It should be []Token, but we don't have parsing for that yet.
+	pool       *ConnPool
+	status     *nodeStatus
+}
+
+func (n *Node) SetStatus(status frame.StatusChangeType) {
+	switch status {
+	case frame.Up:
+		n.status.Store(statusUP)
+	case frame.Down:
+		n.status.Store(statusDown)
+	default:
+		log.Fatalf("status change not supported: %s", status)
+	}
+}
+
+func (n *Node) GetStatus() bool {
```

Comment:
If we have a function `GetStatus()` shouldn't we have, even not exported one, storeStatus?

```go
func (n *Node) storeStatus(status bool) {
    n.status.Store(status)
}
```
to keep consistency?